src/test/scala/edu/ie3/simona/service/ev/ExtEvDataServiceSpec.scala

```diff
@@ -639,13 +642,13 @@ class ExtEvDataServiceSpec
       // we trigger ev service
       scheduler.send(evService, Activation(tick))
 
-      val evsMessage1 = evcs1.expectMsgType[ProvideEvDataMessage]
+      val evsMessage1 = evcs1.expectMsgType[DataProvision[_]]
```

Comment:
As I understood this will be done by DataProvision instead. But wouldn't be there a need to specify the data type?

Reply:
I decided to eliminate all `Provide*DataMessage`, because it didn't give us any benefit. The type of data is still specified by the type parameter `D <: Data`.